### ISSUE TITLE
Spark Launch Method does not work with devel

### ISSUE THREAD

**Issue by iparask** [radical-cybertools/radical.pilot#1307]
Hello,

with ```fix/issue_1305``` launching a Spark cluster is fixed. The CU launching script though is empty.

This is how it looked after testing for #1306 :

```
[iparask@comet-ln2 unit.000000]$ cat radical_pilot_cu_launch_script.sh
#!/bin/sh


# Change to working directory for unit
cd /home/iparask/radical.pilot.sandbox/rp.session.js-172-81.jetstream-cloud.org.iparask.017260.0002/pilot.0000/unit.000000
# Environment variables
export RP_SESSION_ID=rp.session.js-172-81.jetstream-cloud.org.iparask.017260.0002
export RP_PILOT_ID=pilot.0000
export RP_AGENT_ID=agent_1
export RP_SPAWNER_ID=agent.executing.0.child
export RP_UNIT_ID=unit.000000
```

**Comment by iparask** [comment]
Here is the error:

```
2017-04-04 08:56:28,962: agent.executing.0.child: agent.executing.0               : MainThread     : DEBUG   : Created launch_script: /home/iparask/radical.pilot.sandbox/rp.session.js-172-81.jetstream-cloud.org.iparask.017260.0002/pilot.0000/unit.000000/radical_pilot_cu_launch_script.sh
2017-04-04 08:56:28,963: agent.executing.0.child: agent.executing.0               : MainThread     : ERROR   : Error in spawner ('workdir')
Traceback (most recent call last):
  File "/home/iparask/radical.pilot.sandbox/rp.session.js-172-81.jetstream-cloud.org.iparask.017260.0002/pilot.0000/rp_install/lib/python2.7/site-packages/radical/pilot/agent/executing/popen.py", line 278, in spawn
    launch_command, hop_cmd = launcher.construct_command(cu, launch_script_name)
  File "/home/iparask/radical.pilot.sandbox/rp.session.js-172-81.jetstream-cloud.org.iparask.017260.0002/pilot.0000/rp_install/lib/python2.7/site-packages/radical/pilot/agent/lm/spark.py", line 176, in construct_command
    work_dir     = cu['workdir']
KeyError: 'workdir'
2017-04-04 08:56:28,965: agent.executing.0.child: agent.executing.0               : MainThread     : ERROR   : error running CU
```

**Comment by andre-merzky** [comment]
Ah, right - `workdir` is now consistently named `sandbox`.  I'll add that the compatibility nodes. I saw #1308 meanwhile, makes sense :)

**Comment by iparask** [comment]
I removed some unused code.

**Comment by andre-merzky** [comment]
This is now solved via #1307 and #1308.
